code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -213,15 +213,15 @@
 		/obj/item/ammo_casing/shotgun,
 		/obj/item/ammo_magazine,
 		/obj/item/chems/food/snacks/donut/,
-		/obj/item/melee/baton,
-		/obj/item/melee/telebaton,
+		/obj/item/baton,
+		/obj/item/telebaton,
 		/obj/item/flame/lighter,
 		/obj/item/flashlight,
 		/obj/item/modular_computer/pda,
 		/obj/item/radio/headset,
 		/obj/item/hailer,
 		/obj/item/megaphone,
-		/obj/item/melee,
+		/obj/item,
```

Comment:
This is probably a little too permissive

Reply:
Hurk I thought I got all of these. Will update.